modules/release-notes/ref-release-notes-breaking-changes.adoc

```diff
@@ -7,7 +7,7 @@ This section lists breaking changes in {product} {product-version}.
 [id="removed-functionality-rhidp-4572"]
 == Updated monitoring and logging metrics
 
-Prom-client metrics have been removed and replaced with OpenTelemetry metrics. As a result, the metrics port has changed from `7007` to `9464`. Deprecated metrics have also been removed. If you had dependencies on these, ensure your prometheus queries are updated. For further information, see link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html-single/monitoring_and_logging/index#assembly-rhdh-observability[Monitoring and logging]
+Prom-client metrics have been removed and replaced with OpenTelemetry metrics. As a result, the metrics port has changed from `7007` to `9464`. Deprecated metrics have also been removed. If you had dependencies on these, ensure your prometheus queries are updated. For further information, see link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{product-version}/html-single/monitoring_and_logging/index[Monitoring and logging]
```

Comment:
release notes are generated using JIRA as the single source of truth. Please apply your changes in https://issues.redhat.com/browse/RHIDP-4572 then run this script:

https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/blob/release-1.4/modules/release-notes/single-source-release-notes.py